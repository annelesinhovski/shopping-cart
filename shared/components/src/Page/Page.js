import React from "react";

import { Container } from "../Container/Container";

function Page({ children, title, ...props }) {
  return (
    <Container title={title} {...props}>
      {children}
    </Container>
  );
}

export { Page };
