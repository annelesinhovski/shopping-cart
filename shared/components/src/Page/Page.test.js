import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { Page } from './Page';

const wrapper = (props) => render(<Page {...props}>Teste</Page>);

test('Matches snapshot', () => {
  const { asFragment } = wrapper({ title: 'Exemplo de titulo' });

  expect(asFragment()).toMatchSnapshot();
});
