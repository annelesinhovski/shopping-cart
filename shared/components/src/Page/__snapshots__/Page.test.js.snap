// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Containerstyles__Wrapper-sc-11vv4hz-0 eptceT"
  >
    Teste
  </div>
</DocumentFragment>
`;
